src/Definition/ConfigProcessor.php

```diff
@@ -6,7 +6,7 @@
 
 use Overblog\GraphQLBundle\Definition\ConfigProcessor\ConfigProcessorInterface;
 
-final class ConfigProcessor implements ConfigProcessorInterface
```

Comment:
ConfigProcessor don't have to implement ConfigProcessorInterface, because this class is a container for the **real** config processors. Only the 3 real config processors should implement this interface.